### ISSUE TITLE
Provide a bill-of-materials pom.xml

### ISSUE THREAD

**Issue by rovarga** [woq-blended/akka-osgi#6]
Usability of bundles produced by this project would greatly be improved if there was an 'akka-osgi-bom' artifact that could be imported by downstreams:
```xml
<dependencyManagement>
  <dependency>
    <groupId>de.wayofquality.blended</groupId>
    <artifactId>akka-osgi-bom</artifactId>
    <version>2.6.6.1</version>
    <type>pom</type>
    <scope>import</scope>
  </dependency>
</dependencyManagement>
```
to acquire a consistent set of artifacts from a particular release.

**Comment by atooni** [comment]
Hello, is that a pom referencing all the currently supported akka-jars ? Sorry to ask a stupid question, but we are not using maven ourselves, if you have created such a thing yourself for your project, perhaps you could attach it ?

**Comment by rovarga** [comment]
Yes, that would be a declaration of all artifacts produced from this project.

In ODL we are using a non-canonical name, for example:
https://search.maven.org/artifact/org.opendaylight.yangtools/yangtools-artifacts/5.0.3/pom
Netty provides one, too:
https://search.maven.org/artifact/io.netty/netty-bom/4.1.49.Final/pom

It then becomes very easy to keep consistency between projects:
https://github.com/opendaylight/odlparent/blob/master/odlparent/pom.xml#L490-L498
and then a simple reference to the artifact can be used (this place inherits that import):
https://github.com/opendaylight/odlparent/blob/master/features/odl-netty-4/pom.xml#L35-L38
